docs/Manual/07 Plans - Vectorization and Parallelization.md

```diff
@@ -73,9 +73,9 @@ The dimension marked for vectorization is of size 8, which is a supported vector
   00000001400010C3: 48 83 E8 01        sub         rax,1
   00000001400010C7: 75 E7              jne         00000001400010B0
 ```
-Note how the multiplication instruction *vmulps* and the memory move instruction *vmovups* deal with 8 32-bit floating point values at a time.
+Note how the multiplication instruction *vmulps* and the memory move instruction *vmovups* deal with *eight* 32-bit floating-point values at a time.
```

Comment:
I like the usage of "eight" here. Perhaps we can just use "eight" without the * emphasizing it.